src/test/java/com/sparta/greg/pom/pages/ConsultancySkills.java

```diff
@@ -7,12 +7,12 @@
     WebDriver webDriver;
 
     public enum buttonsOnTheConsultancySkills {
```

Comment:
Sorry Sidhant, I meant the whole word should be capitalised. Like this:

1.6. enum naming convention

By convention, enums are constants. In Java, constants are defined in all UPPER_CASE letters. This follows are enums also.

enum name should be in title case (same as class names).
enum fields should be in all UPPER CASE (same as static final constants).